cloud-config/oam-server-api.yml

```diff
@@ -26,5 +63,13 @@ coreos:
         Restart=always
 
         ExecStartPre=-/usr/bin/docker kill oam-server-publisher
-        ExecStart=/usr/bin/docker run --rm --name oam-server-publisher quay.io/hotosm/oam-server-publisher:v0.1.0
+        ExecStart=/usr/bin/docker run \
+          --rm \
+          --name oam-server-publisher \
+          -e 'AWS_DEFAULT_REGION={aws_region}' \
+          -e 'OAM_STATUS_SQS_QUEUE_URL={status_sqs_queue_url}' \
+          -e 'OAM_API_TOKEN={oam_api_token}' \
+          -e 'OAM_STATUS_BUCKET={status_bucket}' \
+          -e 'OAM_STATUS_PREFIX={status_prefix}' \
+          quay.io/hotosm/oam-server-api:{publisher_version}
```

Comment:
Not sure if the responsibilities of the services has changed, but you have this container named as `oam-server-publisher`, but are using the `oam-server-api` container image.